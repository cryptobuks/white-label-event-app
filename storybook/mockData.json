{
  "id": "scheduleItem",
  "author":   {
    "id": 8,
    "name": "Cedric Devroey",
    "imageUrl": "https://picsum.photos/300/250/?random"
  },
  "title": "Bridging the Gap",
  "location": "Arena",
  "date": "Tue Mar 06 2018 16:22:31 GMT+0100 (CET)",
  "labelColor": "pink",
  "labelValue": "Label"
}
