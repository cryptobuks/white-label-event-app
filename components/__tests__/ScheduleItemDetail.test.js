import React from 'react';
import renderer from 'react-test-renderer';
import { ScheduleItemDetail } from '../';
import mockData from '../../assets/mockData.json';

describe('ScheduleItemDetail', () => {
  it('renders correctly', () => {
    const comp = (
      <ScheduleItemDetail
        location={mockData.location}
        date={new Date('Tue Mar 06 2018 14:22:31 GMT+0100 (CET)')}
      />
    );

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
