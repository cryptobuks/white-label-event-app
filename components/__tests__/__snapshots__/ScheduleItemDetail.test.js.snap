// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleItemDetail renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-end",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginTop": 32,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "height": 32,
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#424749",
          "fontSize": 12,
        }
      }
    >
      Arena
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#e54cfe",
          "fontSize": 12,
        }
      }
    >
      02:22 PM
    </Text>
  </View>
  <Image
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    source={1}
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
    testID={undefined}
  />
</View>
`;
